layout-server/src/main/java/org/finos/vuu/layoutserver/dto/response/LayoutResponseDTO.java

```diff
@@ -7,8 +7,6 @@
 @Data
 public class LayoutResponseDTO {
 
-    private UUID id;
```

Comment:
I have removed the id from here as it is already in "metadata": 
![image](https://github.com/ScottLogic/finos-vuu/assets/98337074/731733a2-58b7-4bc1-8ff1-d688e97bcdda)

Reply:
I am happy to add id back in

Reply:
Done!